lms/static/sass/views/_login-register.scss

```diff
@@ -509,16 +509,16 @@
         }
 
         &.button-oa2-azuread-oauth2 {
-            color: darken($microsoft-blue, 20%);
+            color: darken($microsoft-black, 20%);
```

Comment:
how about we remove the darken and just use the black as in the requirement they have asked for black color ?